src/community/ogcapi/ogcapi-features/src/main/java/org/geoserver/ogcapi/v1/features/FeatureServiceXStreamPersisterInitializer.java

```diff
@@ -0,0 +1,20 @@
+package org.geoserver.ogcapi.v1.features;
```

Comment:
Lacks copyright header (QA must be run manually, the module is still community here)